### ISSUE TITLE
[Bug]: 'index_not_found_exception' while trying to use team's knowledge base

### ISSUE THREAD

**Issue by EzeLLM** [infiniflow/ragflow#4039]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### RAGFlow workspace code commit ID

765a114b

### RAGFlow image version

v0.13

### Other environment information

```Markdown
os: ubuntu
```

### Actual behavior

# Knowledge Base Integration Error

## Issue Description
Getting a 404 error while attempting to use the team's knowledge base integration. The error is present in chat and search's llm answer part. The search is successful and returning good results, though.
Issue is not present when the knowledge base is used by the owner. 

### Error Message

`ERROR: NotFoundError(404, 'index_not_found_exception', 'no such index [ragflow_027c5596ba3a11ef8b0e581122d96340]', ragflow_027c5596ba3a11ef8b0e581122d96340, index_or_alias)
`
## Environment Details
- MinIO credentials are consistent across:
  - `.env` file
  - `service_conf.yml`
- MinIO status: Green (6.1ms response time) in `profile > system`

## Troubleshooting Steps Taken

1. Verified MinIO configuration consistency (as per issue #219)
2. Attempted manual index creation in Elasticsearch
   - Result: Error disappears
   - New Issue: LLM cannot access knowledge base
   - Symptom: Always returns 'answer not found in knowledge base'
3. Performed complete system reset:
   - Deleted all containers
   - Deleted all volumes
   - Ran `docker compose` again
   - Result: Issue persists

## Current Status
- Issue remains unresolved
- Manual index creation works but breaks LLM functionality
- System reset did not resolve the problem

## Additional Context
The error appears to be related to Elasticsearch index initialization, suggesting potential issues with:
- Index creation during system startup
- Permissions/access configuration
- Service initialization order

### Expected behavior

- Knowledge base should be accessible to all team members
- No 404 errors should appear
- LLM should properly integrate with knowledge base

### Steps to reproduce

```Markdown
#### Steps to Reproduce

1. **Initial Setup Verification**
   - Confirm MinIO credentials in `.env` file
   - Verify matching credentials in `service_conf.yml`
   - Check MinIO status in `profile > system` (should show green)

2. **Knowledge Base Access Test**
   - Log in as knowledge base owner
   - Attempt to access knowledge base (should work)
   - Log in as team member
   - Attempt same access (should produce 404 error)

3. **Error Details**
   - Monitor for specific error message:
     
     ERROR: NotFoundError(404, 'index_not_found_exception', 
     'no such index [ragflow_027c5596ba3a11ef8b0e581122d96340]', 
     ragflow_027c5596ba3a11ef8b0e581122d96340, index_or_alias)
     ```

4. **System State**
   - Confirm search functionality works
   - Verify LLM answer component fails
   - Note that index is missing for non-owner users

5. **Attempted Solutions**
   - Manual index creation in Elasticsearch
     - Note temporary error resolution
     - Document LLM access failure
     - Record "answer not found in knowledge base" error

6. **System Reset Attempt**
   - Stop all containers
   - Remove containers
   - Delete all volumes
   - Run `docker compose` fresh
   - Verify issue persistence

#### Expected Behavior
- Knowledge base should be accessible to all team members
- No 404 errors should appear
- LLM should properly integrate with knowledge base

#### Actual Behavior
- 404 errors for non-owner access
- Manual index creation breaks LLM functionality
- System reset does not resolve the issue
```


### Additional information

WARN[0000] The "HF_ENDPOINT" variable is not set. Defaulting to a blank string. 
WARN[0000] The "MACOS" variable is not set. Defaulting to a blank string. 
[+] Running 5/0
 ✔ Container ragflow-minio   Created                                                                                                                               0.0s 
 ✔ Container ragflow-redis   Created                                                                                                                               0.0s 
 ✔ Container ragflow-es-01   Created                                                                                                                               0.0s 
 ✔ Container ragflow-mysql   Created                                                                                                                               0.0s 
 ✔ Container ragflow-server  Created                                                                                                                               0.0s 
Attaching to ragflow-es-01, ragflow-minio, ragflow-mysql, ragflow-redis, ragflow-server
ragflow-redis   | 1:C 14 Dec 2024 16:52:14.750 # WARNING Memory overcommit must be enabled! Without it, a background save or replication may fail under low memory condition. Being disabled, it can also cause failures without low memory condition, see https://github.com/jemalloc/jemalloc/issues/1328. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
ragflow-redis   | 1:C 14 Dec 2024 16:52:14.750 * oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
ragflow-redis   | 1:C 14 Dec 2024 16:52:14.750 * Redis version=7.2.4, bits=64, commit=00000000, modified=0, pid=1, just started
ragflow-redis   | 1:C 14 Dec 2024 16:52:14.750 * Configuration loaded
ragflow-redis   | 1:M 14 Dec 2024 16:52:14.750 * monotonic clock: POSIX clock_gettime
ragflow-redis   | 1:M 14 Dec 2024 16:52:14.751 * Running mode=standalone, port=6379.
ragflow-redis   | 1:M 14 Dec 2024 16:52:14.751 * Server initialized
ragflow-redis   | 1:M 14 Dec 2024 16:52:14.751 * Loading RDB produced by version 7.2.4
ragflow-redis   | 1:M 14 Dec 2024 16:52:14.751 * RDB age 33 seconds
ragflow-redis   | 1:M 14 Dec 2024 16:52:14.751 * RDB memory usage when created 1.27 Mb
ragflow-redis   | 1:M 14 Dec 2024 16:52:14.751 * Done loading RDB, keys loaded: 3, keys expired: 0.
ragflow-redis   | 1:M 14 Dec 2024 16:52:14.751 * DB loaded from disk: 0.000 seconds
ragflow-redis   | 1:M 14 Dec 2024 16:52:14.751 * Ready to accept connections tcp
ragflow-mysql   | 2024-12-14 19:52:14+03:00 [Note] [Entrypoint]: Entrypoint script for MySQL Server 8.0.39-1.el9 started.
ragflow-minio   | MinIO Object Storage Server
ragflow-minio   | Copyright: 2015-2023 MinIO, Inc.
ragflow-minio   | License: GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
ragflow-minio   | Version: RELEASE.2023-12-20T01-00-02Z (go1.21.5 linux/amd64)
ragflow-minio   | 
ragflow-minio   | Status:         1 Online, 0 Offline. 
ragflow-minio   | S3-API: http://#########:9000  http://127.0.0.1:9000   
ragflow-minio   | Console: http://#########:9001 http://127.0.0.1:9001  
ragflow-minio   | 
ragflow-minio   | Documentation: https://min.io/docs/minio/linux/index.html
ragflow-minio   | Warning: The standard parity is set to 0. This can lead to data loss.
ragflow-mysql   | 2024-12-14 19:52:14+03:00 [Note] [Entrypoint]: Switching to dedicated user 'mysql'
ragflow-mysql   | 2024-12-14 19:52:14+03:00 [Note] [Entrypoint]: Entrypoint script for MySQL Server 8.0.39-1.el9 started.
ragflow-mysql   | '/var/lib/mysql/mysql.sock' -> '/var/run/mysqld/mysqld.sock'
ragflow-mysql   | 2024-12-14T16:52:15.255893Z 0 [Warning] [MY-011068] [Server] The syntax '--skip-host-cache' is deprecated and will be removed in a future release. Please use SET GLOBAL host_cache_size=0 instead.
ragflow-mysql   | 2024-12-14T16:52:15.256764Z 0 [Warning] [MY-010918] [Server] 'default_authentication_plugin' is deprecated and will be removed in a future release. Please use authentication_policy instead.
ragflow-mysql   | 2024-12-14T16:52:15.256780Z 0 [System] [MY-010116] [Server] /usr/sbin/mysqld (mysqld 8.0.39) starting as process 1
ragflow-mysql   | 2024-12-14T16:52:15.263007Z 1 [System] [MY-013576] [InnoDB] InnoDB initialization has started.
ragflow-mysql   | 2024-12-14T16:52:15.508183Z 1 [System] [MY-013577] [InnoDB] InnoDB initialization has ended.
ragflow-minio   | 
ragflow-minio   |  You are running an older version of MinIO released 10 months before the latest release 
ragflow-minio   |  Update: Run `mc admin update` 
ragflow-minio   | 
ragflow-minio   | 
ragflow-mysql   | 2024-12-14T16:52:15.700363Z 0 [Warning] [MY-010068] [Server] CA certificate ca.pem is self signed.
ragflow-mysql   | 2024-12-14T16:52:15.700387Z 0 [System] [MY-013602] [Server] Channel mysql_main configured to support TLS. Encrypted connections are now supported for this channel.
ragflow-mysql   | 2024-12-14T16:52:15.705169Z 0 [Warning] [MY-011810] [Server] Insecure configuration for --pid-file: Location '/var/run/mysqld' in the path is accessible to all OS users. Consider choosing a different directory.
ragflow-mysql   | 2024-12-14T16:52:15.731843Z 0 [System] [MY-011323] [Server] X Plugin ready for connections. Bind-address: '::' port: 33060, socket: /var/run/mysqld/mysqlx.sock
ragflow-mysql   | 2024-12-14T16:52:15.731886Z 0 [System] [MY-010931] [Server] /usr/sbin/mysqld: ready for connections. Version: '8.0.39'  socket: '/var/run/mysqld/mysqld.sock'  port: 3306  MySQL Community Server - GPL.
ragflow-mysql   | 2024-12-14T16:52:15.738955Z 9 [Warning] [MY-013360] [Server] Plugin mysql_native_password reported: ''mysql_native_password' is deprecated and will be removed in a future release. Please use caching_sha2_password instead'
ragflow-es-01   | Dec 14, 2024 7:52:16 PM sun.util.locale.provider.LocaleProviderAdapter <clinit>
ragflow-es-01   | WARNING: COMPAT locale provider will be removed in a future release
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:17.288Z", "log.level": "INFO", "message":"Java vector incubator API enabled; uses preferredBitSize=512", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.apache.lucene.internal.vectorization.PanamaVectorizationProvider","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:17.572Z", "log.level": "INFO", "message":"version[8.11.3], pid[138], build[docker/64cf052f3b56b1fd4449f5454cb88aca7e739d9a/2023-12-08T11:33:53.634979452Z], OS[Linux/6.8.0-49-generic/amd64], JVM[Oracle Corporation/OpenJDK 64-Bit Server VM/21.0.1/21.0.1+12-29]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.node.Node","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:17.573Z", "log.level": "INFO", "message":"JVM home [/usr/share/elasticsearch/jdk], using bundled JDK [true]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.node.Node","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:17.573Z", "log.level": "INFO", "message":"JVM arguments [-Des.networkaddress.cache.ttl=60, -Des.networkaddress.cache.negative.ttl=10, -Djava.security.manager=allow, -XX:+AlwaysPreTouch, -Xss1m, -Djava.awt.headless=true, -Dfile.encoding=UTF-8, -Djna.nosys=true, -XX:-OmitStackTraceInFastThrow, -Dio.netty.noUnsafe=true, -Dio.netty.noKeySetOptimization=true, -Dio.netty.recycler.maxCapacityPerThread=0, -Dlog4j.shutdownHookEnabled=false, -Dlog4j2.disable.jmx=true, -Dlog4j2.formatMsgNoLookups=true, -Djava.locale.providers=SPI,COMPAT, --add-opens=java.base/java.io=org.elasticsearch.preallocate, -Des.cgroups.hierarchy.override=/, -XX:+UseG1GC, -Djava.io.tmpdir=/tmp/elasticsearch-8263675200893344913, --add-modules=jdk.incubator.vector, -XX:+HeapDumpOnOutOfMemoryError, -XX:+ExitOnOutOfMemoryError, -XX:HeapDumpPath=data, -XX:ErrorFile=logs/hs_err_pid%p.log, -Xlog:gc*,gc+age=trace,safepoint:file=logs/gc.log:utctime,level,pid,tags:filecount=32,filesize=64m, -Xms3849m, -Xmx3849m, -XX:MaxDirectMemorySize=2018508800, -XX:G1HeapRegionSize=4m, -XX:InitiatingHeapOccupancyPercent=30, -XX:G1ReservePercent=15, -Des.distribution.type=docker, --module-path=/usr/share/elasticsearch/lib, --add-modules=jdk.net, --add-modules=ALL-MODULE-PATH, -Djdk.module.main=org.elasticsearch.server]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.node.Node","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.166Z", "log.level": "INFO", "message":"loaded module [repository-url]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.166Z", "log.level": "INFO", "message":"loaded module [rest-root]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.166Z", "log.level": "INFO", "message":"loaded module [x-pack-core]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.166Z", "log.level": "INFO", "message":"loaded module [x-pack-redact]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.166Z", "log.level": "INFO", "message":"loaded module [ingest-user-agent]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.166Z", "log.level": "INFO", "message":"loaded module [x-pack-async-search]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.166Z", "log.level": "INFO", "message":"loaded module [x-pack-monitoring]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.167Z", "log.level": "INFO", "message":"loaded module [repository-s3]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.167Z", "log.level": "INFO", "message":"loaded module [x-pack-analytics]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.167Z", "log.level": "INFO", "message":"loaded module [x-pack-ent-search]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.167Z", "log.level": "INFO", "message":"loaded module [x-pack-autoscaling]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.167Z", "log.level": "INFO", "message":"loaded module [lang-painless]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.167Z", "log.level": "INFO", "message":"loaded module [x-pack-ml]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.167Z", "log.level": "INFO", "message":"loaded module [legacy-geo]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.167Z", "log.level": "INFO", "message":"loaded module [lang-mustache]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.167Z", "log.level": "INFO", "message":"loaded module [x-pack-ql]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.167Z", "log.level": "INFO", "message":"loaded module [rank-rrf]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.168Z", "log.level": "INFO", "message":"loaded module [analysis-common]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.168Z", "log.level": "INFO", "message":"loaded module [transport-netty4]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.168Z", "log.level": "INFO", "message":"loaded module [aggregations]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.168Z", "log.level": "INFO", "message":"loaded module [ingest-common]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.168Z", "log.level": "INFO", "message":"loaded module [frozen-indices]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.168Z", "log.level": "INFO", "message":"loaded module [x-pack-identity-provider]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.168Z", "log.level": "INFO", "message":"loaded module [x-pack-shutdown]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.168Z", "log.level": "INFO", "message":"loaded module [x-pack-text-structure]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.168Z", "log.level": "INFO", "message":"loaded module [snapshot-repo-test-kit]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.168Z", "log.level": "INFO", "message":"loaded module [ml-package-loader]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.169Z", "log.level": "INFO", "message":"loaded module [kibana]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.169Z", "log.level": "INFO", "message":"loaded module [constant-keyword]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.169Z", "log.level": "INFO", "message":"loaded module [x-pack-logstash]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}
ragflow-es-01   | {"@timestamp":"2024-12-14T16:52:19.169Z", "log.level": "INFO", "message":"loaded module [x-pack-ccr]", "ecs.version": "1.2.0","service.name":"ES_ECS","event.dataset":"elasticsearch.server","process.thread.name":"main","log.logger":"org.elasticsearch.plugins.PluginsService","elasticsearch.node.name":"es01","elasticsearch.cluster.name":"docker-cluster"}

**Comment by JinHai-CN** [comment]
Several team related issues are fixed in 0.14.0 and later version. Would you please use try latest version, such as nightly build?

**Comment by KevinHuSh** [comment]
It's usually caused by ES failure.
Check the logs while chunking.
Make sure the free disk space if up to 20%.
